contracts/libraries/GnosisAuction.sol

```diff
@@ -53,8 +54,12 @@ library GnosisAuction {
             IERC20(auctionDetails.oTokenAddress).balanceOf(address(this))
         );
 
+        // minBidAmount is total oTokens to sell * premium per oToken
+        // shift decimals to correspond to decimals of USDC for puts
+        // and underlying for calls
         uint256 minBidAmount =
-            auctionDetails.oTokenPremium.mul(oTokenSellAmount);
+            dswmul(oTokenSellAmount.mul(10**12), auctionDetails.oTokenPremium)
+                .div(10**(uint256(18).sub(auctionDetails.assetDecimals)));
```

Comment:
I'm trying to understand the math here. Why do we multiply 10**12 for the otoken amounts? Shouldn't it be 10**10 since otoken is 8 decimal places